### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- fixed composer in CI
- updated composer dev deps
- fix errors reported by PHPStan 
- clean up in code

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ env:
   extensions: "json"
   cache-version: "1"
   composer-version: "v2"
-  composer-install: "composer update --no-interaction --no-progress --no-suggest --prefer-dist --prefer-stable"
```
Comment (paveljanda) [inline .github/workflows/main.yaml:17]: Just for info: why are we removing this?

File: src/Queue/Queue.php [hunk 2]
```diff
@@ -13,32 +13,32 @@ final class Queue implements IQueue
...
 
 	/**
-	 * @var bool
+	 * @ var bool
 	 */
 	/*private $noWait;*/
 
 	/**
-	 * @var array
+	 * @ var array
```
Comment (paveljanda) [inline src/Queue/Queue.php:41]: This seems weird. Why is there a space?

File: .github/workflows/main.yaml (hunk 1 above)
Comment (petrparolek) [reply .github/workflows/main.yaml:17]: because this 
obrazek

Comment (paveljanda) [reply .github/workflows/main.yaml:17]: Nice, thanks!

File: src/Queue/Queue.php (hunk 2 above)
Comment (petrparolek) [reply src/Queue/Queue.php:41]: PHPStan means that is iregular PHP code and it reports error.

Comment (paveljanda) [reply src/Queue/Queue.php:41]: This is a normal PHP annotation https://php-annotations.readthedocs.io/en/stable/UsingAnnotations.html

Comment (petrparolek) [reply src/Queue/Queue.php:41]: But code is commented.

Comment (paveljanda) [reply src/Queue/Queue.php:41]: I see. God knows why. Give me a minute.

Comment (paveljanda) [reply src/Queue/Queue.php:41]: I know why. Queue properties get declared separately. Could you, please, remove all this dead code? 🙌

Comment (petrparolek) [reply src/Queue/Queue.php:41]: OK

Comment (petrparolek) [reply src/Queue/Queue.php]: Done

**Comment by paveljanda** [comment]
Thanks

**Pull request merged**